### PR TITLE
[ROC-661] update_retention_eligibility_definition

### PR DESCRIPTION
**Pull request by wangy70**

### CODE REVIEW

File: rdr_service/model/participant_summary.py [hunk 1]
```diff
@@ -73,7 +73,11 @@
 _COMPUTE_RETENTION_ELIGIBLE_SQL = """
     CASE WHEN
       consent_for_study_enrollment = 1
-      AND (consent_for_electronic_health_records = 1 OR consent_for_dv_electronic_health_records_sharing = 1)
+      AND (
+        consent_for_electronic_health_records = 1 OR
+        ehr_consent_expire_status = 2 OR
+        consent_for_dv_electronic_health_records_sharing = 1
+      )
```
Comment (kenny-skaggs) [inline rdr_service/model/participant_summary.py:79]: Shouldn't participants remain retention eligible even if they've changed their EHR consent from Yes to No?

Comment (wangy70) [reply rdr_service/model/participant_summary.py:79]: Good catch, I will make the change.

Comment (wangy70) [reply rdr_service/model/participant_summary.py:79]: Done

Comment (kenny-skaggs) [reply rdr_service/model/participant_summary.py:79]: 👍

**Comment by kenny-skaggs** [comment]
Just notice one more thing: the migration will need to base itself off of the cb2e353a154f migration from PR #1976.

**Review by kenny-skaggs — APPROVED** [review]

**Pull request merged**